src/components/App.jsx

```diff
@@ -104,6 +105,7 @@ function PageContent({
       <main>
         <Switch>
           <Route path="/orders" component={ConnectedOrderHistoryPage} />
+          <Route path="/wip-payments" component={ConnectedPaymentsPage} />
```

Comment:
why not just /payments to start?

Reply:
I added a TODO to rename everything to `payment`.

Reply:
Oops. I think I missed the point of your question. Did you mean: why include wip? Not a big deal either way, but just making it really clear. We can ditch it when we move this to the new repo.